src/js/components/Notification/Notification.js

```diff
@@ -45,38 +94,74 @@ const Notification = ({ message, onClose, id, status, title, toast }) => {
   ]);
 
   const { icon: CloseIcon } = theme.notification.close;
-  const { icon: StatusIcon, color } = theme.notification[status];
+  const { icon: StatusIcon, background, color } = theme.notification[status];
   const { color: closeIconColor } = theme.notification.close;
 
+  const { direction, truncate } = toast
+    ? theme.notification.toast
+    : theme.notification;
+
+  const TextWrapper = direction === 'row' ? Text : Box;
```

Comment:
Any reason to not always put inside a `Box`?

Reply:
We need the whole thing to be in text for the truncation to work properly (otherwise the title and message won't truncate as a single string, which made for strange wrapping behavior). However, we can leave the whole thing nested in another Box as it was prior.